.github/ISSUE_TEMPLATE/instrumentation_request.md

```diff
@@ -1,41 +1,36 @@
 ---
 name: Instrumentation request
-about: Create a report to add support for an instrumentation
+about: Create a request to add a new instrumentation
 labels: instrumentation-request
 ---
 
 <!--
-**NB:** Before opening an instrumentation request against this repo, please read [the contributing guidelines for new instrumentation](../../CONTRIBUTING.md#new-instrumentation).
+**NB:** Before opening an instrumentation request against this repo, please read [the contributing guidelines for new instrumentation](../blob/main/CONTRIBUTING.md#new-instrumentation).
 -->
 
-### Is your instrumentation request related to a problem? Please describe
+### Which library do you want to instrument?
 <!--
-A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
+A clear and concise description of why this library should be instrumented. Ex. I'm always frustrated when [...]
 -->
 
-
-### Is it applicable for Node or Browser or both?
-
-
-### Do you expect this instrumentation to be commonly used?
-
-Weekly Downloads:
-
-### What version of instrumentation are you interested in using?
-
-Versions:
-
 ### Code Owners
 
 <!--
-New Instrumentation requests need at least 2 code-owners that will maintain the instrumentation long-term.
+New Instrumentations MUST have at least 2 code-owners that will maintain the instrumentation long-term.
+Any code-owners listed MUST fulfill all criteria laid out in the checklist below.
 -->
 
 - Owner 1
+  - [ ] I am a [member of the OpenTelemetry Organization](https://github.com/open-telemetry/community/blob/main/guides/contributor/membership.md#member)
+  - [ ] I have read [CONTRIBUTING.md](../blob/main/CONTRIBUTING.md) and understand the responsibilities of a component owner
+  - [ ] I agree to follow and uphold the [mission, vision and values](https://github.com/open-telemetry/community/blob/main/mission-vision-values.md) of the OpenTelemetry project
+  - [ ] I understand that the component may be subject to the [OpenTelemetry Semantic Conventions](https://github.com/open-telemetry/semantic-conventions) and agree to follow the rules set out therein
 - Owner 2
+  <!-- instruct any co-owners to complete the checklist above and comment it on this issue -->
+  - [ ] Commented the checklist below and has fulfilled the criteria for becoming a component owner
```

Comment:
Haha, yes I was confused when writing it too and then gave up at some point 😬 
Thank you for the suggestion - I think your version is better 🙂 

```suggestion
  - [ ] Commented on the issue confirming they have fulfilled all criteria for becoming a component owner
```